webapp/src/blocks.tsx

```diff
@@ -561,8 +561,6 @@ export class Editor extends toolboxeditor.ToolboxEditor {
         const blocklyDiv = this.getBlocksEditorDiv();
         // Position blocklyDiv over blocklyArea.
         if (blocklyDiv && this.editor) {
-            blocklyDiv.style.width = blocklyArea.offsetWidth + 'px';
```

Comment:
@shakao does this look alright / reasonable (along with changes in blockly-core.less: https://github.com/microsoft/pxt/pull/7045/files#diff-f2432c11e75a6cd6a1d5f64674a06542R23)? It was the quick workaround for some resizing bugs (so I didn't have to duplicate the size logic all over the place), but it feels like it must have been done this way for a reason..

Easy enough to fix and maintain this if needed (e.g. if svgResize peeks at the width maybe? haven't pulled blockly on this machine to check quickly)

Reply:
this looks good to me! accessing the .width property will still return the correct measurement with this moved to css